test/integration/suites/agent-cli/06-check-api-watch-fail

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+SVID_RECEIVED=1
+TIMEOUT_REACHED=0
+
+# Run the background process and store its output in a temporary file
+(docker-compose exec -u 1001 -T spire-agent /opt/spire/bin/spire-agent api watch < /dev/null > api_watch_output.txt) &
+
+# Get the PID of the last background process
+API_WATCH_PID=$!
+
+# Continuously check the output file for the desired pattern with a timeout of 20 seconds
+TIMEOUT=20
+START_TIME=$(date +%s)
+while ! grep -q "Received 1 svid after" api_watch_output.txt; do
+    CURRENT_TIME=$(date +%s)
+    ELAPSED_TIME=$((CURRENT_TIME - START_TIME))
+    if [ $ELAPSED_TIME -gt $TIMEOUT ]; then
+        echo "Error: Timeout reached while waiting for 'Received' message."
```

Comment:
you can use `fail-now` function here to finish the process with error

Reply:
In this test we're expecting that the message isn't received, because we don't have an entry.
Maybe should I clarify that?

Reply:
yes, but is it a cause of IT failure right?